<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainDialog</class>
 <widget class="QDialog" name="mainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mapbender QGIS-Plugin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="editSectionComboBoxLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Please select the server configuration section to edit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="editSectionComboBox"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="editServiceNameLabel">
       <property name="text">
        <string>Service name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="editServerAddressLabel">
       <property name="text">
        <string>Server address</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="editPortLabel">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="editUserNameLabel">
       <property name="text">
        <string>User name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="editPasswordLabel">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editServiceNameLineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editServerAddressLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editPortLineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="editUserNameLineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="editPasswordLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="saveEditedSectionButton">
     <property name="text">
      <string>Save changes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="removeDialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
